### PR TITLE
config: enable tichi for `pingcap/tiunimanager` and `pingcap/tiunimanager-ui`

### PR DESCRIPTION
**Pull request by wuhuizuo**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

### CODE REVIEW

File: prow/config/config.yaml [hunk 1]
```diff
@@ -1498,6 +1498,22 @@ branch-protection:
                   - 'license/cla'
                   - 'idc-jenkins-ci-tispark/integration-test'
                 strict: true
+      tiunimanager:
+        branches:
+          main:
+            protect: true
+            required_status_checks:
+              contexts:
```
Comment (Mini256) [inline prow/config/config.yaml:1506]: Maybe `jenkins-tiem/verify` need to be required?

cc: @haiboumich, please confirm it.

File: prow/config/config.yaml [hunk 2]
```diff
@@ -1498,6 +1498,22 @@ branch-protection:
...
+                - "tide"
+                - "license/cla"
+      tiunimanager-ui:
+        branches:
+          main:
+            protect: true
+            required_status_checks:
+              contexts:
+                - "tide"
```
Comment (Mini256) [inline prow/config/config.yaml:1515]: Should GitHub Action `build` and `lint` passed before PR merged?

cc: @Yuiham

File: prow/config/external_plugins_config.yaml [hunk 3]
```diff
@@ -1353,6 +1357,8 @@ ti-community-tars:
       - pingcap/tiflash
       - pingcap/tiflash-scripts
       - pingcap/tispark
+      - pingcap/tiunimanager
```
Comment (Mini256) [inline prow/config/external_plugins_config.yaml:1360]: The `pingcap/tiunimanager` didn't use the Jenkins CI, so we don't need to `/run-all-tests` after the bot performed the merge-base operation.

**Review by Mini256 — COMMENTED** [review]
Please add `ti-community-owners` plugin config, we need it to provide permission information for other plugins.

File: prow/config/config.yaml (hunk 1 above)
Comment (haiboumich) [reply prow/config/config.yaml:1506]: > Maybe `jenkins-tiem/verify` need to be required?
> 
> cc: @haiboumich, please confirm it.

confirm

**Review by haiboumich — APPROVED** [review]
LGTM

**Review by Mini256 — APPROVED** [review]

File: prow/config/external_plugins_config.yaml [hunk 4]
```diff
@@ -418,6 +422,18 @@ ti-community-owners:
       - bigdata-reviewers
     sig_endpoint: https://bots.tidb.io/ti-community-bot
     require_lgtm_label_prefix: require-LGT
+  - repos:
+      - pingcap/tiunimanager
+      - pingcap/tiunimanager-ui
+    default_require_lgtm: 2
```
Comment (Yuiham) [inline prow/config/external_plugins_config.yaml:428]: `default_require_lgtm` for tiunimanager-ui should be 1.

**Review by Yuiham — CHANGES_REQUESTED** [review]

File: prow/config/external_plugins_config.yaml [hunk 5]
```diff
@@ -418,6 +422,18 @@ ti-community-owners:
       - bigdata-reviewers
     sig_endpoint: https://bots.tidb.io/ti-community-bot
     require_lgtm_label_prefix: require-LGT
+  - repos:
+      - pingcap/tiunimanager
+      - pingcap/tiunimanager-ui
+    default_require_lgtm: 1
```
Comment (Yuiham) [inline prow/config/external_plugins_config.yaml:428]: Will this change affect the `tiunimanager` repo?

**Pull request merged**